independent-publisher-connectors/OpenCage Geocoding/apiProperties.json

```diff
@@ -0,0 +1,24 @@
+{
+  "properties": {
+    "connectionParameters": {
+      "api_key": {
+        "type": "securestring",
+        "uiDefinition": {
+          "displayName": "API Key",
+          "description": "The API Key for this api",
+          "tooltip": "Provide your API Key",
+          "constraints": {
+            "tabIndex": 2,
+            "clearText": false,
+            "required": "true"
+          }
+        }
+      }
+    },
+    "iconBrandColor": "#007ee5",
```

Comment:
@ahmadn82 Please update it to the default iconBrandColor of all the IP connectors.

`"iconBrandColor": "#da3b01"`